packages/common/package.json

```diff
@@ -45,6 +45,11 @@
   },
   "dependencies": {
     "@configu/sdk": "workspace:*",
+    "@configu-integrations/validator": "workspace:*",
+    "@configu-integrations/comptact-json": "workspace:*",
```

Comment:
Should be `compact` instead of `comptact`

Reply:
fixed.